src/commands/build.ts

```diff
@@ -71,7 +73,7 @@ export default defineCommand({
           target: 'esnext',
         },
         emitCJS: false,
-        cjsBridge: true,
+        cjsBridge: false,
```

Comment:
This also disables ability to have __filename / __dirname from ESM files litle bit dangerous we should clearly document migration if we disable

Reply:
Jiti in dev also allows this so modules only fail in production published

Reply:
will make sure to document ❤️

Reply:
didn't see the last comment - do you think it is better to keep?

Reply:
i think made it few mins after when remembered. more streict eslint rules might help but that's just a fact that module authors might miss until too late. disabling is fine too considering we eventually want to drop it.